silverback/_cli.py

```diff
@@ -707,7 +712,7 @@ def cancel_payment_stream(
         )
 
     elif cluster.status != ResourceStatus.RUNNING:
```

Comment:
Actually come to think of it, we don't want to gatekeep cancelling like this, the stream logic will prevent people from cancelling it at bad times